packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -2193,7 +2193,19 @@ class BaseModelSqlv2 {
         await this.beforeInsert(insertObj, trx, cookie);
       }
 
-      await this.model.getColumns();
+      const columns = await this.model.getColumns();
+
+      // exclude auto increment columns in body
+      for (const col of columns) {
+        if (col.ai === true) {
```

Comment:
could be just `if (col.ai) {` since in some cases it will be `1` ( in some database it stored as tinyint(1) )

Reply:
ok